src/views/virtualmachines/actions/components/VirtualMachineMigration/hooks/migrateBulkVMs.ts

```diff
@@ -0,0 +1,105 @@
+import { IoK8sApiCoreV1PersistentVolumeClaim } from '@kubevirt-ui/kubevirt-api/kubernetes';
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { getName, getNamespace } from '@kubevirt-utils/resources/shared';
+import { getRandomChars } from '@kubevirt-utils/utils/utils';
+import { k8sCreate, k8sGet, k8sPatch } from '@openshift-console/dynamic-plugin-sdk';
+
+import { MigMigration, MigMigrationModel, MigPlan, MigPlanModel } from '../constants';
+
+const getEmptyMigPlan = (namespace: string): MigPlan => ({
```

Comment:
why Mig? :)

Reply:
this is the resource name `MigPlan` from MTC